plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -7686,4 +7533,20 @@ private GetVmVncTicketAnswer execute(GetVmVncTicketCommand cmd) {
             return new GetVmVncTicketAnswer(null, false, e.getLocalizedMessage());
         }
     }
-}
+
+    private String createLogMessageException(Throwable e, Command command) {
```

Comment:
possible to move this method to VMware helper class and use wherever applicable.

observed some usages here:

- https://github.com/apache/cloudstack/blob/8a16729fcf10d9e6952fcccc07d8d17952fa6312/plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java#L725-L733

- https://github.com/apache/cloudstack/blob/8a16729fcf10d9e6952fcccc07d8d17952fa6312/plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java#L897-L904

- https://github.com/apache/cloudstack/blob/8a16729fcf10d9e6952fcccc07d8d17952fa6312/plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java#L2048-L2055